app/scripts/modules/core/notification/modal/editNotification.controller.modal.js

```diff
@@ -63,6 +63,10 @@ module.exports = angular
       $uibModalInstance.close(vm.notification);
     };
 
+    vm.supportsCustomMessage = function(notification) {
```

Comment:
I don't believe so — the method is invoked based on the state of the notification type select box which has no empty option.